src/interpreter/plugin/LookupPlugin.ts

```diff
@@ -12,7 +12,7 @@ import {ProcedureAst} from '../../parser'
 import {StatType} from '../../statistics'
 import {zeroIfEmpty} from '../ArithmeticHelper'
 import {InterpreterState} from '../InterpreterState'
-import {InternalScalarValue, InterpreterValue, RawNoErrorScalarValue, } from '../InterpreterValue'
+import {InternalScalarValue, InterpreterValue, RawNoErrorScalarValue,} from '../InterpreterValue'
```

Comment:
Linter reported problem here
```suggestion
import {InternalScalarValue, InterpreterValue, RawNoErrorScalarValue} from '../InterpreterValue'
```